editor-open.cabal

```diff
@@ -37,9 +37,13 @@ library
   build-depends:
      base ==4.8.*
```

Comment:
I would really like to support an older version of `base`, for people on Debian/Ubuntu/other-distros-with-moldy-software.

I'm not suggesting you do anything, it's just a comment.